lib/index.js

```diff
@@ -1,21 +1,50 @@
 var AWS = require('aws-sdk');
 AWS.config.region = process.env.AWS_REGION;
+
+
+
+function set_http_proxy() {
+    var proxySet = false;
+    var proxy = require('proxy-agent');
```

Comment:
@Nelsonic it's required for enterprise that use corporate proxy, i made it optional that if you don't provide it, nothing will break and your code will continue as it is, I will take it out  , i can add this in actual app code.